packages/office-ui-fabric-react/src/components/__snapshots__/DetailsList.Grouped.Example.tsx.shot

```diff
@@ -777,6 +777,7 @@ exports[`Component Examples renders DetailsList.Grouped.Example.tsx correctly 1`
                   }
               data-is-focusable={true}
               onClick={[Function]}
+              role="button"
```

Comment:
This snapshot might be stale since it has `role="button"`, but no corresponding `aria-label`. Perhaps re-generate if so.

Reply:
Nevermind, I see it at https://github.com/OfficeDev/office-ui-fabric-react/pull/10056/files#diff-e853a1df429e869eb91fdc969a30e91dL729.

Thanks @mashiau!